uttut/tests/test_expand_by_entities.py

```diff
@@ -41,34 +48,46 @@ def setUp(self):
         ]
         self.expected_entity_lists = [
             [
-                Entity(name='日期', value='明天', start=3, end=5, replacements=['下禮拜二']),
+                Entity(name='日期', value='明天', start=3,
+                       end=5, replacements=['下禮拜二']),
```

Comment:
Avoid coding style that is inconsistent with our repo

Reply:
e1ac824